docker_deep_dive_part_1/Deploy_Wordpress_With_Multiple_Container.md

```diff
@@ -0,0 +1,66 @@
+## Deploy Wordpress With Multiple Container 
+In this section we deploy WordPress Website With Multiple Container
+
+Start off by making a new directory where you wish to store the files for WordPress and MariaDB for example in your home directory.
+```
+mkdir ~/wordpress && cd ~/wordpress
+```
+
+### Docker Setup For Database Server 
+
+Search Mariadb Image
+```
+docker search mariadb
+```
+
+Pull mariadb docker image
+```
+docker pull mariadb
+```
+
+Run This Database Container 
+* run -- use for run the container
+* -e -- for the env variable like in below command we use MYSQL_ROOT_PASSWORD env variable to assign password
+* –name wordpress -- Gives the container a name.
+* -v “$PWD/database”:/var/lib/mysql -- Creates a data directory linked to the container storage to ensure data persistence.
+* -d -- Tells Docker to run the container in daemon.
+* mariadb:latest -- name of image
+```
+docker run -e MYSQL_ROOT_PASSWORD=root123 -e MYSQL_DATABASE=wordpress --name wordpressdb -v "$PWD/database":/var/lib/mysql -d mariadb:latest
+```
+
+List running docker processes 
+```
+docker ps
+```
+
+
+### Docker Setup For Wordpress
+
+Search Wordpress Image
+```
+docker search wordpress
```

Comment:
Same as above

Reply:
i use that command is just to people aware about that command also.(before pull they can check that which images able related to that search) .